examples/decoding/plot_receptive_field.py

```diff
@@ -0,0 +1,156 @@
+
+.. _figure 1: http://journal.frontiersin.org/article/10.3389/fnhum.2016.00604/full#F1
+.. _figure 2: http://journal.frontiersin.org/article/10.3389/fnhum.2016.00604/full#F2
+"""  # noqa: E501
+
+# Authors: Chris Holdgraf <choldgraf@gmail.com>
+#
+# License: BSD (3-clause)
+# sphinx_gallery_thumbnail_number = 3
+
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.io import loadmat
+from scipy.stats import pearsonr
+
+import mne
+from mne.decoding import ReceptiveField
+from sklearn.linear_model import Ridge
+from sklearn.model_selection import KFold
+from sklearn.preprocessing import scale
+
+
+# Create a correlation sklearn-style scorer
+def corr_score(y_true, y, multioutput=None):
+    return [pearsonr(y_true[:, ii], y[:, ii])[0] for ii in range(y.shape[-1])]
+
+
+###############################################################################
+# Load the data from the publication
+# ----------------------------------
+#
+# First we will load the data collected in [1]_. In this experiment subjects
+# listened to natural speech. Raw EEG and the speech stimulus was collected.
+# We will load these below, downsampling the data in order to speed up
+# computation since we know that our features are primarily low-frequency in
+# nature. Then we'll visualize both the EEG and speech envelope.
+
+path = mne.datasets.mtrf.data_path()
+n_decim = 2
+data = loadmat(path + '/speech_data.mat')
+raw = data['EEG'].T
+speech = data['envelope'].T
+sfreq = float(data['Fs'])
+sfreq /= n_decim
+speech = mne.filter.resample(speech, down=n_decim, npad='auto')
+raw = mne.filter.resample(raw, down=n_decim, npad='auto')
+
+
+# Read in channel positions and create our MNE objects from the raw data
+montage = mne.channels.read_montage('biosemi128')
+montage.selection = montage.selection[:128]
+info = mne.create_info(montage.ch_names[:128], sfreq, 'eeg', montage=montage)
+raw = mne.io.RawArray(raw, info)
+n_channels = len(raw.ch_names)
+
+# Plot a sample of brain and stimulus activity
+fig, ax = plt.subplots()
+ax.plot(scale(raw[:][0][..., :800].T), color='k', alpha=.1)
+ax.plot(scale(speech[0, :800]), color='r', lw=2)
+ax.set(title="Sample brain and stimulus activity", xlabel="Time (s)")
+mne.viz.tight_layout()
+
+###############################################################################
+# Create and fit a receptive field model
+# --------------------------------------
+#
+# We will construct a model to find the linear relationship between the EEG
+# signal and a time-delayed version of the speech envelope. This allows
+# us to make predictions about the response to new stimuli.
+
+# Define the delays that we will use in the receptive field
+tmin, tmax = -.4, .2
+
+# Initialize the model
+spec_names = ['freq_%s' % ii for ii in range(speech.shape[0])]
+rf = ReceptiveField(tmin, tmax, sfreq, feature_names=spec_names,
+                    estimator=Ridge(alpha=1.), scorer=corr_score)
```

Comment:
sklearn uses scoring as param to specify a score.

git grep "scorer="

returns nothing in sklearn code base